modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/NavigationButtons.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayIcon from '@clayui/icon';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const NavigationButtons = ({
+	moveDown,
+	moveLeft,
+	moveRight,
+	moveUp,
+	spritemap,
+}) => {
+	const navigationButtonStyle = {
```

Comment:
This can be moved outside of this function as a constant or moved inline of the jsx element